FIPS/public_pom.xml

```diff
@@ -32,8 +32,8 @@
     </scm>
 
     <properties>
-      <bouncycastle.bcfips.version>1.0.2.4</bouncycastle.bcfips.version>
-      <bouncycastle.bcpkixfips.version>1.0.5</bouncycastle.bcpkixfips.version>
+      <bouncycastle.bcfips.version>1.0.2.5</bouncycastle.bcfips.version>
```

Comment:
The `1.0.2.5` version is not FIPS-validated. The only version that's higher than current and has a valid FIPS certificate is `2.0.0` so I guess for FIPS customers it'll make sense to migrate directly to this version

Reply:
We cannot upgrade to 2.0.0 since it does not support TLSv1.2 correctly.
We discuss internally that it's ok to use version 1.0.2.5